backend/base-backend-package/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@bufferapp/base-backend-package",
+  "version": "0.0.7",
+  "description": "A base package for writing new backend packages.",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "files": [
+    "lib/**/*"
+  ],
+  "scripts": {
+    "build": "tsc",
+    "format": "prettier --write \"src/**/*.ts\"",
+    "lint": "eslint . --ext .ts",
+    "test": "jest",
+    "prepare": "npm run build",
+    "prepublishOnly": "npm test && npm run lint",
+    "preversion": "npm run lint",
+    "version": "npm run format && git add -A src",
+    "postversion": "git push . HEAD && git push --tags"
```

Comment:
This does not work as expected 🤔 It does not create a commit, and also with a repo with several packages, how will tags work?

Reply:
hmm it should tag with the version defined in `version` for each individual package (each package should have it's own package.json). Shall we have a script to run `npm publish` here ? 🤔  I'm not sure to understand the meaning of `postversion` here

Reply:
Yeah, from [this blog post](https://itnext.io/step-by-step-building-and-publishing-an-npm-typescript-package-44fe7164964c), it is supposed to commit and push to GH automatically. But, it may be missing the `git commit` command.